### ISSUE TITLE
Reuse

### ISSUE THREAD

**Issue by Edelweiss** [DCLP/dclpxsltbox#259]
In the **editor**

Checkbox that tells whether the piece of papyri has been reused or not (yes/no)
Free text where the editor can give information about what type of reuse
Free text where the editor can give a reference to an already existing idp.data record (e.g. ddbdp or another DCLP record)

Find appropriate **TEI/EpiDoc** xpaths where to store this information

suggestion for possible xpath for type
```xpath
tei:physDesc/tei:…
```
xpath for reference pointer

```xpath
tei:physDesc/tei:…/tei:ptr/@target (http://papyri.info/ddbdp/xyz;;1234)
```

**Comment by paregorios** [comment]
@wsalesky @leoba @hcayless have any of you seen in other TEI projects the sort of reuse notes/indicators called out by @Edelweiss above? If so, where/how have they been encoded?

**Comment by leoba** [comment]
I have a bit of experience thinking about palimpsests, but usually that's thinking about how to encode palimpsestic texts and not describing the reuse of the parchment (or papyri) itself. I found this discussion from last year on the TEI listserv, which doesn't have an answer to this query, but shows other people are thinking about it (and realizing the TEI doesn't deal well with reuse): http://tei-l.970651.n3.nabble.com/palimpsest-td4028621.html

It seems to me that if we're talking about whether a papyrus has been reused we're talking about it's condition, so I'd use tei:condition. tei:condition can contain tei:ptr. So the code would look something like:

`<tei:condition>palimpsest <ptr target="http://papyri.info/ddbdp/xyz;;1234"/></tei:condition>`

Condition is part of tei:supportDesc:

`tei:physDesc/tei:objectDesc/tei:supportDesc/tei:condition`

I did a search of the existing XSLT and the only references to tei:condition I found are in dclpxsltbox/xslt/pn-xslt/metadata.xsl; in the XML I didn't find any references to it, so at first glance it seems to be available for this use.

**Comment by paregorios** [comment]
Thanks @leoba. I think ```<condition>``` is a good locus for this information. I've searched through the papyrological data, and the only collection in which that element is used is APIS, where it's a catch-all container within ```<supportDesc>```. 

I think we probably need something pretty strictly structured for indicating the fact (or not) of re-use. Sometimes we won't know both the before and after, and in other case one or the other text will not be available digitally or even published at all. I talked this aspect through with @gabrielbodard and @simonastoyanova this morning. They suggested creating a ```<category>``` element within ```encodingDesc/classDecl/taxonomy``` to contain the necessary distinction, and then pointing at it with the ```@ana``` attribute on ```<condition>```. Something like:

```xml
<category xml:id="reused">
    <catDesc>A support that has been re-used</catDesc>
</category>
```

They also suggested using the ```@corresp``` attribute, instead of an ```<ptr>```, to indicate related text(s).

What does everyone think about this hybrid suggested approach?

**Comment by leoba** [comment]
Yes to both creating categories in the taxonomy section, and to using @corresp instead of <ptr>.

**Comment by paregorios** [comment]
@ Edelweiss, do items 1 and 2 below seem like a good way to accomplish the agenda you set above? 

1. Within ```/TEI/teiHeader/encodingDesc``` and immediately following ```<p>``` therein, add the following static subtree:

```xml
<classDecl>
   <taxonomy>
      <category xml:id="reused">
         <catDesc>A support that has been re-used</catDesc>
      </category>
   </taxonomy>
</classDecl>
```

2. When a papyrus is reused, add the following child to ```/TEI/teiHeader/fileDesc/sourceDesc/msDesc/physDesc/objectDesc/supportDesc```:

```xml
<condition @ana="#reused"></condition>
```

I would like (after @leoba's suggestion above) also to propose (attn @rla2118 @jcowey @HolgerEssler @rogerbagnall) that -- in addition to a simple yes/no for reuse, which the above provides -- we also give ourselves the option to:

3. Add an optional textual note regarding the character etc. of the reuse, e.g.:

```xml
<condition @ana="#reused">The papyrus is clearly reused; however, the original ink has been so effectively removed that the character of the original text cannot be determined.</condition>
```

4. Add the ability optionally to link to other entries that provide the prior text(s), e.g.:

```xml
<condition @ana="#reused" @corresp="http://papyri.info/ddbdp/bgu;1;57 http://litpap.info/dclp/60044">The papyrus is clearly reused; however, the original ink has been so effectively removed that the character of the original text cannot be determined.</condition>
```

NB: appropriate affordance in the editor would be required as well for nos. 3 and 4. 

Thoughts on 3 and 4?

**Comment by rogerbagnall** [comment]
Yes, both 3 and 4 seem potentially useful to me.

**Comment by paregorios** [comment]
@Edelweiss can you comment on how difficult it would be to implement support for suggestions 3 and 4, above, in the editor mask? cc @jcowey @rla2118 @HolgerEssler

**Comment by ryanfb** [comment]
Related to #301, it seems like whatever XML is decided on should have a corresponding XSLT in `pn-mapping` for the RDF conversion, so that the relation gets loaded into the triplestore (numbers server).

**Comment by paregorios** [comment]
I am still waiting for a response to my question above regarding difficulty of implementation in the editor. Until we have that response, this ticket cannot move forward. Moreover, it should now be considered to be blocking #301.

**Comment by paregorios** [comment]
@jcowey @rla2118 can you help me get an answer to my question, above?

**Comment by paregorios** [comment]
@jcowey @rla2118 This has emerged as a significant concern of participants in the New York workshop.

**Comment by Edelweiss** [comment]
I don’t think implementing a re-use feature for the editor would be a huge amount of work. I estimate finding the appropriate xpaths a far greater task. Thanks for going to the trouble of finding meaningful and viable encoding strategies. As for the editor, all we need to do is add the respective xpaths to the dclp configuration and provide some helper functions and GUI elements.

Are there any ideas as to what the new feature should look like? We likely need a checkbox that can be ticked if an object has been re-used, a text area for comments and a list of URLs of corresponding documents.

Under which category do you want the new editor feature to reside? Physical Description, General Notes, Bibliography or perhaps even a new category called ‘Re-use’ or ‘Palimpsest?’

**Comment by paregorios** [comment]
It seems to me now that the next step would be for the papyrologists ( @rla2118 , @rogerbagnall , @jcowey  and colleagues ) to come up with answers to the questions about appearance and user interface raised in the second and third paragraphs of @Edelweiss's [immediately preceding comment on this issue](https://github.com/DCLP/dclpxsltbox/issues/259#issuecomment-336845255).

**Comment by jcowey** [comment]
Have already spoken to Carmen concerning this. I think that our priority should be to fix the Create New Publication business (#326 , #330 etc), then go through the process of getting that deployed. 
Thereafter we can return to this.

**Comment by jds15** [comment]
We have access to TM re-use data, via tab-sep dump provided by Mark. It is *not* automatically updated. So, it does not in current implementation provide solution to all desired use cases (e.g. papyri.info user wants to associate two or more o/w unassociated texts). But it will surely be an assent if we mean to include re-use data somewhere in the records. 

This means that this is a UI issue and also an EpiDoc issue. And if the idea is to handle all such re-use in same fashion also a "mine TM and touch all implicated records" issue. So, valuable, definable, and non-trivial.

Probably best IMO to focus on "Create New Pub", which is a blocker for basic function.

**Comment by rla2118** [comment]
I also think Create New Pub should be the priority.

2017-10-16 16:23 GMT+02:00 jds15 <notifications@github.com>:
